diesel/src/expression/helper_types.rs

```diff
@@ -8,6 +8,8 @@ pub type SqlTypeOf<Expr> = <Expr as Expression>::SqlType;
 pub type AsExpr<Item, TargetExpr> = AsExprOf<Item, SqlTypeOf<TargetExpr>>;
 pub type AsExprOf<Item, Type> = <Item as AsExpression<Type>>::Expression;
 
+#[macro_export]
```

Comment:
This name is too generic for us to export. You shouldn't need to export it to use it from another module.